engine.py

```diff
@@ -178,17 +178,20 @@ def execute_command(self, cmd_key, args):
                 # We need to clear Qt library paths on Linux if KDE is the active environment.
                 # This resolves issues with mismatched Qt libraries between the OS and the
                 # application being launched if it is a DCC that comes with a bundled Qt.
+                # It appears to only need to be fixed in PySide (1), PySide2 is fine.
                 if (
                     tank.util.is_linux()
                     and os.environ.get("KDE_FULL_SESSION") is not None
+                    and QtCore.qVersion()[0] == 4
```

Comment:
```suggestion
                    and QtCore.qVersion()[0] == "4"
```
It's a string. Nice testing bro. 😜

Reply:
Oh for goodness sake! I did test it, but obviously didn't copy the code over correct from my vm to my mac. Sorry about that, thanks.